Sources/NIOHTTP2/HTTP2FrameEncoder.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  HTTP2FrameEncoder.swift
+//  NIOHTTP2
+//
+//  Created by Jim Dovey on 6/27/18.
+//
+
+import Foundation
```

Comment:
I think we can drop this. :wink:

Reply:
Oops. That shouldn't have been in the commit 🤫

Reply:
Still here ;)